bftengine/src/bftengine/MsgsCommunicator.hpp

```diff
@@ -15,6 +15,7 @@
 #include "communication/ICommunication.hpp"
 #include "IncomingMsgsStorage.hpp"
 #include "messages/IncomingMsg.hpp"
+#include "diagnostics.h"
```

Comment:
Why do you need to add this in this hpp if you havn't changed anything in it?
If some other file doesnt compile you should consider adding it (if needed) there.